src/stories/InteractiveStepSubHeader.stories.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, {useRef} from 'react';
+import {Button, View} from 'react-native';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
+const story = {
+    title: 'Components/InteractiveStepSubHeader',
+    component: InteractiveStepSubHeader,
+};
+
+function Template(args) {
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    return <InteractiveStepSubHeader {...args} />;
+}
+
+// Arguments can be passed to the component by binding
+// See: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Default = Template.bind({});
+
+function BaseInteractiveStepSubHeader(props) {
+    const ref = useRef(null);
+    return (
+        <View>
+            <InteractiveStepSubHeader
+                {...props}
```

Comment:
Probably `// eslint-disable-next-line react/jsx-props-no-spreading` should be added

Reply:
no needed, stories have different eslint rules